### PR TITLE
compactor: Add version 0.10.1

### LINKED ISSUE #7726: [Request] Compactor
### Information

**Name:** compactor

**Description:** GUI frontend for Windows' `compact.exe`

**Homepage:** https://github.com/Freaky/Compactor

**Some indication of popularity/repute:** 610 stars

Made a PR for it: #7725

### PR DESCRIPTION
**Pull request by couleurm**
close #7726

### CODE REVIEW

File: bucket/compactor.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "0.10.1",
+    "description": "A user interface for Windows 10 filesystem compression",
+    "homepage": "https://github.com/Freaky/Compactor",
+    "license": "MIT",
+    "architecture": {
+        "64bit":{
+            "url": "https://github.com/Freaky/Compactor/releases/download/v0.10.1/Compactor-0.10.1.7z",
+            "hash": "sha256:1dc75fd5dad212e0cbf29a363632e0299945183d722b2c00e527e54ed86de379",
```
Comment (rashil2000) [inline bucket/compactor.json:9]:
```suggestion
            "hash": "1dc75fd5dad212e0cbf29a363632e0299945183d722b2c00e527e54ed86de379",
```

File: bucket/compactor.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            "extract_dir": "Compactor-0.10.1"
+        },
+        "32bit":{
+            "url": "https://github.com/Freaky/Compactor/releases/download/v0.10.1/Compactor-0.10.1-i686.7z",
+            "hash": "sha256:015e358422db0eeb4d71305aaa4a74991f43c46074e26d6a9487645ab109083b",
```
Comment (rashil2000) [inline bucket/compactor.json:14]:
```suggestion
            "hash": "015e358422db0eeb4d71305aaa4a74991f43c46074e26d6a9487645ab109083b",
```

File: bucket/compactor.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            "extract_dir": "Compactor-0.10.1-i686"
+        }
+    },
+    "bin": "Compactor.exe",
```
Comment (rashil2000) [inline bucket/compactor.json:18]: Commandline arguments?

File: bucket/compactor.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    "shortcuts": [
+        [
+            "Compactor.exe",
+            "Compactor"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/Freaky/Compactor"
+    },
```
Comment (rashil2000) [inline bucket/compactor.json:27]:
```suggestion
    "checkver": "github",
```

File: bucket/compactor.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    "autoupdate": {
+        "architecture": {
+            "64bit":{
+                "url": "https://github.com/Freaky/Compactor/releases/download/v$ver/Compactor-$ver.7z"
```
Comment (rashil2000) [inline bucket/compactor.json:31]:
```suggestion
                "url": "https://github.com/Freaky/Compactor/releases/download/v$version/Compactor-$version.7z",
                "extract_dir": "Compactor-$version"
```

File: bucket/compactor.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+            },
+            "32bit":{
+                "url": "https://github.com/Freaky/Compactor/releases/download/v$ver/Compactor-$ver-i686.7z"
```
Comment (rashil2000) [inline bucket/compactor.json:34]:
```suggestion
                "url": "https://github.com/Freaky/Compactor/releases/download/v$version/Compactor-$version-i686.7z",
                "extract_dir": "Compactor-$version-i686"
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/compactor.json (hunk 3 above)
Comment (couleurm) [reply bucket/compactor.json:18]: I haven't checked, I just got the habit of launching stuff through Run

File: bucket/compactor.json (hunk 5 above)
Comment (couleurm) [reply bucket/compactor.json:31]: any difference between $ver and $version?

File: bucket/compactor.json (hunk 3 above)
Comment (rashil2000) [reply bucket/compactor.json:18]: Please check it

File: bucket/compactor.json (hunk 5 above)
Comment (rashil2000) [reply bucket/compactor.json:31]: $ver is not valid

File: bucket/compactor.json (hunk 3 above)
Comment (couleurm) [reply bucket/compactor.json:18]: it does not, I'll remove it

**Comment by couleurm** [comment]
image
Is this blocking the PR? It keeps asking about one more request but I resolved everything.. 🤔

**Pull request merged**